snippets/visualbasic/VS_Snippets_CLR_System/system.collections.generic.equalitycomparer.default/vb/program.vb

```diff
@@ -1,130 +1,111 @@
-'<Snippet1>
 Imports System
```

Comment:
> It's added automatically by the VB template, but I have no idea why.

Maybe it would make sense to open an issue on [dotnet/templating](https://github.com/dotnet/templating/) about that?